packages/addons/eslint/index.ts

```diff
@@ -85,11 +85,25 @@ export default defineAddon({
 			const globalsNode = common.createSpreadElement(common.expressionFromString('globals.node'));
 			const globalsObjLiteral = object.createEmpty();
 			globalsObjLiteral.properties = [globalsBrowser, globalsNode];
+			const off = common.createLiteral('off');
+			off.comments = [
```

Comment:
Without running prettier, this generates the following code:
![image](https://github.com/user-attachments/assets/95c49171-e193-4cbd-8571-806bfd8f0f00)

The comment probably needs to be added to the `rules` variable or something like this